xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -971,8 +1062,8 @@ void CVideoPlayer::OpenDefaultStreams(bool reset)
       valid = true;
       if(!psp.relevant(stream))
         visible = false;
-      else if(stream.flags & StreamFlags::FLAG_FORCED)
-        visible = true;
+      //else if(stream.flags & StreamFlags::FLAG_FORCED)
+      //  visible = true;
```

Comment:
Hello @CastagnaIT, what was the idea behind this removal? It made sense to force forced subs on when user has subs off by default. Those lines were not responsible for the issue #26050.

Reply:
> what was the idea behind this removal?

I think removal is OK because this condition now is included inside "relevant" filter.

> It made sense to force forced subs on when user has subs off by default.

No, because then the setting options "None" and "Forced only" behaves the same.

If user has subs language = "None" and continues displaying forced subs then is the same than "Forced only".

As it is now is fine.

> Those lines were not responsible for the issue https://github.com/xbmc/xbmc/issues/26050.

Agree. This issue still present in current master.

Reply:
> About subtitles "default" flag

> from what i understood on the subtitles the "default" flag is not so important since it is used only
> to give more priority to a stream, for the case when there are two streams with same properties (lang > and flags, but maybe different author version or codec)
> On kodi there is no “Media default” setting for subs in the GUI, such as happens on audio language > > setting, and there is no intention to implement it

With this PR you have ruined a decades-long behavior, namely that subtitles containing the default flag are automatically displayed

Congratulations on this insane change